doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -692,7 +692,6 @@ Passthrough a Hard Disk to RTVM
          -s 2,passthru,00/17/0 \
          -s 3,virtio-console,@stdio:stdio_port \
          -s 8,virtio-net,tap0 \
-         $pm_channel $pm_by_vuart \
```

Comment:
This line should be removed in the script directly too.

Reply:
@NanlinXie will submitted another PR to delete it in the script;

Reply:
OK, thanks